config/sentry.php

```diff
@@ -54,6 +54,11 @@
         // Capture views as spans
         'views' => true,
 
+        // Capture storage access as spans
+        // Filesystems have to opt-in by changing their configuration like this:
+        // ['driver' => 'local', ...] -> ['driver' => 'sentry', 'original_driver' => 'local', ...]
```

Comment:
Instead of requiring manual configuration, we could do something like this in `StorageIntegration::setup()`:

```php
foreach (config('filesystem.disks') as $disk => $config) {
    config(["filesystem.disks.{$disk}.original_driver" => $config['driver']);
    config(["filesystem.disks.{$disk}.driver" => 'sentry');
}
```

What do you think?

Reply:
This feel very hacky... like very 😅 not sure at this moment.

Reply:
I just tried it in my application and it seems to cause no issues so far, see [`4f322e0` (#726)](https://github.com/getsentry/sentry-laravel/pull/726/commits/4f322e080c14561c62baa893057bd5b378a24680).

Advantages to this approach:
- it just works™
- no manual configuration required
- no errors as mentioned in https://github.com/getsentry/sentry-laravel/pull/726#discussion_r1252182612 are possible

Disadvantages:
- feels scary

Reply:
Yep, I hear ya. Let's keep it like this!